.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Saw that you removed this file in a later commit, please squash those commits with this one so in the git history the files were never added in the first place.